### ISSUE TITLE
get_button in windows/base_view.cpp

### ISSUE THREAD

**Issue by Xeverous** [cycfi/elements#16]
2 things are suspicious here

https://github.com/cycfi/elements/blob/a5724e68ff446a177523f64701d708d68e1667fe/lib/host/windows/base_view.cpp#L185-L259

- the `click_count` is never used (triggers unused variable warning)
  - this is either unneded or something was forgotten
- `mouse_button::what which` is set in the switch but never used (triggers variable set but not used warning)
  - the switch which sets the value of `which` is either unneded or something was forgotten
  - the function always returns `mouse_button::left`, which looks like a potential replacement for `which` variable

**Comment by djowel** [comment]
I suppose you'll add the PR next? I'd like to review the exact affected lines.

**Comment by Xeverous** [comment]
`click_count` can be removed as such change does not change any behavior (no idea what the former author meant with this) but I'm unsure what should be done with `which`. The switch also does not have a default case.

**Comment by djowel** [comment]
Yes, click_count can be removed. Now that I look at it, pardon my confusion with that and -> click_count. I think your changes are good. Ditto for 'which'. Please post the PR.

**Comment by Xeverous** [comment]
How about the switch? It does not have a `default` case and `which` is not given any initial value.

**Comment by djowel** [comment]
We can probably just have mouse_button::left as default. Feel free to suggest some code in your PR.

**Comment by Xeverous** [comment]
#19 is merged
